core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -314,6 +314,13 @@ public enum Property {
       "Enables additional JVM metrics collection and reporting using Micrometer. Requires "
           + "property 'general.micrometer.enabled' to be set to 'true' to take effect.",
       "2.1.0"),
+  GENERAL_MICROMETER_LOG_METRICS_ENABLED("general.micrometer.log.metrics.enabled", "none",
```

Comment:
Could drop the word enabled from the end of the prop name because that word makes it sound like the values should be a boolean.

```suggestion
  GENERAL_MICROMETER_LOG_METRICS("general.micrometer.log.metrics", "none",
```

Reply:
Applied in a043322.